include/qunit.hpp

```diff
@@ -345,11 +351,10 @@ class QUnit : public QInterface {
     template <typename F, typename... B> void EntangleAndCallMember(F fn, B... bits);
     template <typename F, typename... B> void EntangleAndCall(F fn, B... bits);
     template <typename F, typename... B> void EntangleAndCallMemberRot(F fn, real1 radians, B... bits);
+    template <typename F> bool ParallelUnitApply(F fn);
```

Comment:
Ditto for here - the other functions need to accept a variable number of parameters, whereas this doesn't.

Reply:
The comment above notwithstanding, this lambda happens to be reasonably amenable to conversion to avoid capture. I'll push an update.